src/common/net/Listener.cc

```diff
@@ -65,7 +59,7 @@ Result<Void> Listener::setup() {
 
   auto &filters = config_.filter_list();
   for (auto [name, addr] : nics.value()) {
-    if (addr.up && (filters.empty() || filters.count(name) != 0) && checkNicType(name, networkType_)) {
+    if (addr.up && (filters.empty() || filters.count(name) != 0) && checkNicType(networkType_)) {
```

Comment:
consider deleting the `checkNicType` function and replacing `checkNicType(networkType_)` with `networkType_ != Address::UNIX`

Reply:
networkType_ is not get from IfAddrs::load(),intresting

Reply:
in this case ,this fun post "RDMA" to networkType_, NIC only filter by name